app/src/sm_ppp.c

```diff
@@ -56,7 +56,9 @@ enum ppp_action {
 };
 
 enum ppp_reason {
-	PPP_REASON_DEFAULT,
+	PPP_REASON_CMD,
+	PPP_REASON_NETWORK,
+	PPP_REASON_ERROR,
 	PPP_REASON_PEER_DISCONNECTED,
```

Comment:
Short description would be nice related to in which cases these are supposed to be used.